src/pages/App.tsx

```diff
@@ -200,6 +200,12 @@ export default function App() {
     return null
   }
 
+  // redirect from blocked pages if needed
+  const blockedPathsList = document.querySelector('meta[name="cf:blockedpaths"]')?.getAttribute('content')?.split(',')
+  if (blockedPathsList && blockedPathsList?.includes(pathname)) {
+    return <Navigate to="/swap" replace />
+  }
+
```

Comment:
I think this will loop. Instead, can you set this as a variable and use it on L231 (before your edits) with the other redirect?

```suggestion
  const blockedPaths = document.querySelector('meta[name="x:blocked-paths"]')?.getAttribute('content')?.split(',')
  const shouldBlockPath = blockedPaths?.includes(pathname) ?? false
```

Reply:
i added a check to make sure it won't redirect loop - but i don't think your suggestion will work, because it doesn't account for all of the other potential blocked paths - would have to add a unique bool for each possible route